src/IlluminaSampleSheet/V1/Row.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\IlluminaSampleSheet\V1;
+
+use Illuminate\Support\Collection;
+
+abstract class Row
+{
+    public string $sampleID;
+
+    abstract public function toString(): string;
+
+    /** @return Collection<int, string> */
+    abstract public function getColumns(): Collection;
```

Comment:
```suggestion
    abstract public function headerLine(): string;
```
This should simplify it a bit and we do not have any need for structured informations about the header with this implementation, right?